neuro_data/static_images/data_schemas.py

```diff
@@ -892,19 +892,18 @@ class Member(dj.Part):
         name                    : varchar(50) unique # string description to be used for training
         """
 
-    def fill(self):
-
+    @classmethod
```

Comment:
don't think you need the classmethod, you can just make it static and use the name of the class, classmethods are used in very specific scenarios and this could confuse people

Reply:
Fixed